Practice 3

StanislavMolchanovStudentKarazinUa opened the pull request.

StanislavMolchanovStudentKarazinUa committed:
Create practice3.cpp

StanislavMolchanovStudentKarazinUa committed:
Update practice3.cpp: Code from practice2.cpp

I'll commit new changes to practice2.cpp code

StanislavMolchanovStudentKarazinUa committed:
Made practice3.cpp (without header file)

mariiaShvedchenko commented on practice3.cpp:
```diff
@@ -0,0 +1,312 @@
+}
+
+/////////////////////////////////////////////////////////////////////
+bool deleteNode(pnode * head, pnode p) {
```
would be better to notify user somehow that "football player" was successfully or not deleted. so that it would not look like your program went to an infinity loop.

mariiaShvedchenko replied:
and same for other insert functions

mariiaShvedchenko reviewed (commented):
so change a bit your code as per comments. 

everything else LGTM

StanislavMolchanovStudentKarazinUa committed:
Update practice3.cpp: Code redaction#001

Completed requests:
would be better to notify user somehow that "football player" was successfully or not deleted. so that it would not look like your program went to an infinity loop.
and same for other insert functions
Comment:
Notification commands are written in main() function

StanislavMolchanovStudentKarazinUa committed:
Update practice3.cpp : Code redaction#002

-Fixed deleteNode() function
